libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -30,7 +30,7 @@
             <strong>Published:</strong> {{ b.publishedDate | date:'dd/MM/yyyy' }}
           </div>
           <p [innerHTML]="b.description"></p>
-          <div>
+          <div data-testing = "add-to-list">
```

Comment:
I would remove the spaces around the `=` sign after your `data-*` attributes.

Reply:
@mandarini: Review addressed, please take a look.